pyproject.toml

```diff
@@ -106,6 +106,7 @@ fail_under = 100
 show_missing = true
 
 [tool.coverage.run]
+branch = true
```

Comment:
This would result in failing if branch coverage is below `100`, or is there a way to just report on branch coverage without failing?

Reply:
I don’t think so, I poked through the docs and didn’t find anything that indicates your suggestion…